lib/api-blueprint-preview-view.coffee

```diff
@@ -82,8 +82,9 @@ class ApiBlueprintPreviewView extends ScrollView
   renderApiBlueprintText: (text) ->
     fs.writeFileSync '/tmp/atom.apib', text
     # Env hack... helps find aglio binary
-    env =
-        PATH: process.env.PATH + ':/usr/local/bin'
+    env = Object.create(process.env)
+    npm_bin = atom.project.getPaths().map (p) -> "#{path.join(p, 'node_modules', '.bin')}:"
```

Comment:
By “multiple projects” do you mean multiple windows, or multiple root folders in a single project? In Atom one window == one project.

`atom.project.getPaths()` returns a list of root folders for a project. In most cases that would just be a list of one folder. But it is possible to add additional root folders to a project, so it could be an idea to check if the current file to be previewed is located in a sub-path of a given root folder.